src/box/gc.c

```diff
@@ -213,6 +213,11 @@ gc_run_cleanup(void)
 		consumer = gc_tree_next(&gc.consumers, consumer);
 	}
 
+	struct vclock retention_vclock;
+	wal_get_retention_vclock(&retention_vclock);
+	if (vclock_is_set(&retention_vclock))
+		vclock_min_ignore0(&min_vclock, &retention_vclock);
```

Comment:
`retention_vclock` has a non-zero 0-th component, so `min_vclock` should account for it.
We ignore 0-th vclock component only for gc consumers.

Reply:
Sure, let's use `vclock_compare` instead, as there's no `vclock_min` method.

Reply:
> Sure, let's use vclock_compare instead, as there's no vclock_min method.

Why don't you add one?

It seems with `vclock_compare` you may miss the case when vclocks are incomparable.

For example:
`retention_vclock: {1: 1, 2: 2}`
`min_vclock: {1:2, 2: 1}`,
vclocks are incomparable, so `min_vclock` remains as is, even though it should become `{1: 1, 2:1}`.

Reply:
> It seems with vclock_compare you may miss the case when vclocks are incomparable.

Now I see, thank you! Introduced `min_vclock`